### PR TITLE
feat(linked-project-list): display a list of linked projects

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

File: addon/controllers/project.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  @service constructionProject;
+
+  get displayLandingPage() {
+    return (
+      !this.projects.length && this.router.currentRoute.localName !== "new"
+    );
+  }
+
+  get activeProject() {
+    return Number(this.router.currentRoute.params.project_id);
```
Comment (anehx) [inline addon/controllers/project.js:22]:
```suggestion
    return parseInt(this.router.currentRoute.params.project_id, 10);
```

File: addon/controllers/project/form.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import Controller from "@ember/controller";
+import { assert } from "@ember/debug";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { tracked } from "@glimmer/tracking";
+import { task, lastValue } from "ember-concurrency-decorators";
+import Options from "ember-ebau-gwr/models/options";
```
Comment (anehx) [inline addon/controllers/project/form.js:7]: IMO this doesn't belong in the models folder - it's confusing.

File: addon/controllers/project/form.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  @service fetch;
+  @service store;
+  @service router;
+
+  @tracked import = false;
+
+  choiceOptions = Options;
+
+  get currentRoute() {
+    return this.router.currentRoute.name;
```
Comment (anehx) [inline addon/controllers/project/form.js:23]:
```suggestion
    return this.router.currentRouteName;
```

File: addon/templates/project/form.hbs [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+        <div>
+          <button
+            class={{configured-classes "button"}}
+            type="button"
+            {{on "click" this.importCalumaData}}
+          >
+            {{t "ember-gwr.components.projectForm.applyAll"}}
+          </button>
+          <LinkTo
+            @route={{this.currentRoute}}
```
Comment (anehx) [inline addon/templates/project/form.hbs:28]: Can't this be omitted if it's the current route??

File: addon/controllers/project/form.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    this.project.typeOfConstruction = 6223;
+    this.project.projectAnnouncementDate = "11.12.2019";
+    this.project.realestateIdentification.number = 83289;
+    this.project.realestateIdentification.EGRID = 832891;
+    this.project.typeOfClient = 6121;
+    this.project.client.address.street = "Gässli";
+    this.project.client.address.houseNumber = 5;
+    this.project.client.identification.personIdentification.officialName =
+      "Müller";
+    this.project.client.identification.personIdentification.firstName = "Hans";
```
Comment (anehx) [inline addon/controllers/project/form.js:64]: ... okay? Maybe a comment would help me understand what in the world this simulates?

File: addon/controllers/project.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    const projects = yield Promise.all(
+      links.map(({ eproid }) =>
+        this.constructionProject.getFromCacheOrApi(eproid)
+      )
+    );
+
+    // Load the first project in the list if none is selected so we always display a project.
+    if (
+      this.router.currentRoute.localName !== "form" &&
+      this.router.currentRoute.localName !== "new" &&
```
Comment (czosel) [inline addon/controllers/project.js:45]: includes

**Review by anehx — CHANGES_REQUESTED** [review]
I have 0 context in this thing. @czosel please review as well..

File: addon/controllers/search-project.js [hunk 7]
```diff
@@ -71,7 +71,7 @@ export default class SearchProjectController extends Controller {
   linkProject(eproid) {
     const link = this.store.createRecord("gwr-link", {
       eproid,
-      localId: this.model.id,
+      localid: this.model.id,
```
Comment (czosel) [inline addon/controllers/search-project.js:74]: I think this was correct before

Comment (czosel) [reply addon/controllers/search-project.js:74]: model should be fixed as well

**Review by czosel — APPROVED** [review]
:+1:

File: addon/controllers/project/form.js (hunk 5 above)
Comment (czosel) [reply addon/controllers/project/form.js:64]: This is just demo code and will be dropped very soon in favor of the actual implementation, no need I'd say.

**Pull request merged**